src/layout/OrgHeader.tsx

```diff
@@ -0,0 +1,55 @@
+import Link from 'next/link';
+
+interface OrgHeaderProps {
+    org: {
```

Comment:
I think eventually we should type this as `ZetkinOrganization` instead of an object, any object with `id` and `title`.